yt/utilities/answer_testing/framework.py

```diff
@@ -969,6 +983,28 @@ def big_patch_amr(ds_fn, fields, input_center="max", input_weight="density"):
                         dobj_name)
 
 
+def nbody_answer(ds, ds_str_repr, ds_nparticles, fields):
```

Comment:
so if I'm reading this right, this is almost strictly equivalent to the old `sph_answer` routine, with the only difference being the class of the projection test.
Wouldn't it be clearer to rewrite these functions as follow ?

```python
def nbody_answer(ds, ds_str_repr, ds_nparticles, fields,
    projection_value_class=PixelizedParticleProjectionValuesTest):
    ...

def sph_answer(ds, ds_str_repr, ds_nparticles, fields):
    return nbody_answer(ds, ds_str_repr, ds_nparticles, fields,
        projection_value_class=PixelizedProjectionValuesTest)
```

Reply:
yup it would, incoming...